### PR TITLE
Update technologies.md

### PR DESCRIPTION
**Pull request by eric-hc**
Updated technologies

@danielhelfand, @jpunlay please review

### CODE REVIEW

**Comment by danielhelfand** [comment]
@jpunlay Bueno

**Comment by eric-hc** [comment]
@jpunlay isn't committed to [satisfying the customer through early and continuous delivery of valuable software](https://agilemanifesto.org/principles.html), is he @danielhelfand?

**Comment by danielhelfand** [comment]
Should we try and add markdown to bios in this pr as well? Should probably get it in all at once before the end of this sprint.

**Comment by eric-hc** [comment]
I think we have to push this release out.

**Comment by danielhelfand** [comment]
I would like another pair of eyes on this. @julesruggiero, please review this.

File: docs/technologies.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This is a list of the techonologies I am familiar with, not ordered in any parti
     - PostgreSQL
 
 ??? tip "Cloud Environments"
-    - Amazon Web SErvices (AWS)
+    - Amazon Web Services (AWS)
     - OpenShift Origin (OSO)
```
Comment (danielhelfand) [inline docs/technologies.md:33]: Replace (OSO) with (OKD)

Comment (jpunlay) [reply docs/technologies.md:33]: how does k stand for community? As the SME here, please explain

**Comment by jpunlay** [comment]
I need a laptop

File: docs/technologies.md (hunk 1 above)
Comment (eric-hc) [reply docs/technologies.md:33]: Hard K

**Comment by eric-hc** [comment]
He updated technologies and now there is a conflict.

**Comment by danielhelfand** [comment]
Liking the direction here. Please mark this a WIP and let's review again next week.

**Comment by danielhelfand** [comment]
Seems like some of these changes were actually already addressed in [`f8b64d7`](https://github.com/jpunlay/jesuspunlay/commit/f8b64d7291386692e8df466ba56b05ce3df59a20).

**Comment by eric-hc** [comment]
That is why I merged with https://github.com/jpunlay/jesuspunlay/pull/3/commits/60166fd661b18a34adeef28e103f80f2a99f80f5. Please keep up.

**Comment by danielhelfand** [comment]
Technology. Just wow. You move so fast. Be a shame, if you broke something. Like my ❤️.

**Comment by eric-hc** [comment]
Looking for review

**Review by danielhelfand — APPROVED** [review]

**Review by eric-hc — COMMENTED** [review]
I approve

**Review by danielhelfand — APPROVED** [review]

**Comment by eric-hc** [comment]
Going to close this stale PR unless otherwise specified by @jpunlay

**Comment by eric-hc** [comment]
Thank you all for your collaboration on this, I learned a lot.

**Pull request closed without merging**